carservice/views.py

```diff
@@ -0,0 +1,50 @@
+from django.shortcuts import render, redirect
+from django.views import generic, View
+from django.urls import reverse_lazy
+from django.views.generic.edit import CreateView
+
+from carservice.models import Car, Offer, Rent
+
+
+class CarCreateView(CreateView):
+    model = Car
+    fields = ['serial_number', 'car_mileage', 'car_model', 'car_year']
+    template_name = 'car_create.html'
+    success_url = '/car/read'
```

Comment:
taki zahardkodowany url to nic ładnego, lepiej użyć coś `reverse/reverse_lazy`

https://docs.djangoproject.com/en/4.2/ref/urlresolvers/#reverse
https://docs.djangoproject.com/en/4.2/ref/urlresolvers/#reverse-lazy

Reply:
podbijam